### PR TITLE
dev, 3.0: Add tkctl user guide

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

### What is changed, added or deleted? 
Add tkctl user guide

Related PR: https://github.com/pingcap/docs-cn/pull/1420

### Which version does your change affect? 
`dev` `3.0`

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/reference/tools/tkctl.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+- [MacOS](http://download.pingcap.org/tkctl-darwin-amd64-latest.tgz)
+- [Linux](http://download.pingcap.org/tkctl-linux-amd64-latest.tgz)
+- [Windows](http://download.pingcap.org/tkctl-windows-amd64-latest.tgz)
+
+### Build from Source
+
+{{< copyable "shell-regular" >}}
+
+```shell
+git clone https://github.com/pingcap/tidb-operator.git && \
```
Comment (DanielZhangQD) [inline dev/reference/tools/tkctl.md]: Add `--depth=1`?

File: dev/reference/tools/tkctl.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+gdb /proc/${pid:-1}/root/tikv-server 1
+```
+
+{{< copyable "shell-regular" >}}
+
+```shell
+(gdb) set sysroot /proc/${pid}/root/
+```
+
+The `.gdbinit` configured in the `tidb-debug` image will set `sysroot` to `/proc/1/root/` automatically. So, you can omit this following step if you are using the `tidb-debug` image and the `pid` of the target process 1.
```
Comment (DanielZhangQD) [inline dev/reference/tools/tkctl.md]: "this following step" should be "the above step"?

File: dev/reference/tools/tkctl.md [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+
+```
+NAMESPACE NAME           PD    TIKV   TIDB   AGE
+foo       demo-cluster   3/3   3/3    2/2    11m
+bar       demo-cluster   3/3   3/3    1/2    11m
+```
+
+### tkctl use
+
+This command used to specify the current TiDB cluster to use, the other commands could omit `--tidbcluster` option and defaults to select current TiDB cluster if there is a current TiDB cluster set.
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
This command is used to specify the TiDB cluster that the current `tkctl` command operates on. After you specify a TiDB cluster by using this command, all commands that operates on a cluster will automatically select this cluster so the `--tidbcluster` option can be omitted.
```

File: dev/reference/tools/tkctl.md [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+
+Example:
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
For example:
```

File: dev/reference/tools/tkctl.md [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+tkctl use --namespace=foo demo-cluster
+```
+
+```
+Tidb cluster switched to foo/demo-cluster
+```
+
+### tkctl info
+
+This command used to get the information of TiDB cluster, the current TiDB cluster will be used if exists.
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
This command is used to display information about the TiDB cluster.
```

File: dev/reference/tools/tkctl.md [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+
+| Flags | Shorthand | Description |
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
| Flag | Abbreviation | Description |
```

File: dev/reference/tools/tkctl.md [hunk 7]
```diff
@@ -0,0 +1,375 @@
...
+| ----- | --------- | ----------- |
+| --tidb-cluster | -t | select the tidb cluster, default to current TiDB cluster |
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
| --tidb-cluster | -t | Specify a TiDB cluster; default to the TiDB cluster that is being used |
```

File: dev/reference/tools/tkctl.md [hunk 8]
```diff
@@ -0,0 +1,375 @@
...
+
+Example:
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
For example:
```

File: dev/reference/tools/tkctl.md [hunk 9]
```diff
@@ -0,0 +1,375 @@
...
+  TiKV:  Normal   3      3        1000m  2Gi     10Gi     pingcap/tikv:v3.0.0-rc.1
+  TiDB   Upgrade  1      2        500m   1Gi              pingcap/tidb:v3.0.0-rc.1
+Endpoints(NodePort):
+  - 172.16.4.158:31441
+  - 172.16.4.155:31441
+```
+
+### tkctl get [component]
+
+This is a group of commands used to get the details of TiDB cluster componentes, the current TiDB cluster will be used if exists.
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
This is a group of commands that are used to get the details of TiDB cluster components.
```

File: dev/reference/tools/tkctl.md [hunk 10]
```diff
@@ -0,0 +1,375 @@
...
+
+Available components: `pd`, `tikv`, `tidb`, `volume`, `all`(query all components)
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
You can query the following components: `pd`, `tikv`, `tidb`, `volume` and `all` (to query all components).
```

File: dev/reference/tools/tkctl.md [hunk 11]
```diff
@@ -0,0 +1,375 @@
...
+
+| Flags | Shorthand | Description |
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
| Flag | Abbreviation | Description |
```
Update this table accordingly as commented above.

File: dev/reference/tools/tkctl.md [hunk 12]
```diff
@@ -0,0 +1,375 @@
...
+| ----- | --------- | ----------- |
+| --tidb-cluster | -t | select the tidb cluster, default to current TiDB cluster |
+| --output | -o | output format, one of [default,json,yaml], the default format is `default` |
+
+Example:
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
For example:
```

File: dev/reference/tools/tkctl.md [hunk 13]
```diff
@@ -0,0 +1,375 @@
...
+local-pv-5ade8580   tikv-demo-cluster-tikv-1   Bound    1476Gi     172.16.4.160   /mnt/disks/local-pv33
+local-pv-ed2ffe50   tikv-demo-cluster-tikv-2   Bound    1476Gi     172.16.4.157   /mnt/disks/local-pv13
+local-pv-74ee0364   pd-demo-cluster-pd-0       Bound    1476Gi     172.16.4.155   /mnt/disks/local-pv46
+local-pv-842034e6   pd-demo-cluster-pd-1       Bound    1476Gi     172.16.4.158   /mnt/disks/local-pv74
+local-pv-e54c122a   pd-demo-cluster-pd-2       Bound    1476Gi     172.16.4.156   /mnt/disks/local-pv72
+```
+
+### tkctl debug [pod_name]
+
+This command used to diagnose the Pods of TiDB cluster. It launches a debug container for you which has the necessary troubleshooting tools installed.
```
Comment (yikeke) [inline dev/reference/tools/tkctl.md]:
```suggestion
This command is used to diagnose the Pods in a TiDB cluster. It launches a debug container for you which has the necessary troubleshooting tools installed.
```

**Comment by aylei** [comment]
@yikeke I've addressed the review comments and apply the changes to `3.0`, PTAL

**Comment by yikeke** [comment]
I have submitted a commit https://github.com/pingcap/docs/pull/1354/commits/3117433a34f1fd81e52d7ac20d7c1b6838834910 to improve the language and align this PR with the [Chinese version](https://github.com/pingcap/docs-cn/pull/1420/files). Please take a technical review at this commit. @aylei

**Comment by aylei** [comment]
@yikeke LGTM

**Comment by yikeke** [comment]
> @yikeke LGTM

Great! PTAL again. @DanielZhangQD

**Pull request merged**